lib/session.js

```diff
@@ -9,8 +10,32 @@ function Session (opts) {
   }, opts)
 }
 
+Session.prototype.set = function (key, user, cb) {
+  key = normalizeKey(key)
+
+  this.client.hmset(this.sessionLookupPrefix + key, user, function (err, result) {
+    if (err) return cb(errors.error500())
+    return cb(null, result)
+  })
+}
+
+Session.prototype.get = function (key, cb) {
+  key = normalizeKey(key)
+
+  this.client.hgetall(this.sessionLookupPrefix + key, function (err, obj) {
+    if (err) return cb(errors.error500())
+    if (!obj) return cb(errors.error404())
+
+    return cb(null, obj)
+  })
+}
+
 Session.prototype.end = function () {
   this.client.end()
 }
 
+function normalizeKey (token) {
+  return token.replace(/^user-/, '')
```

Comment:
Seems like you could possibly prepend the `sessionLookupPrefix` here before returning, so you wouldn't have to add it in the `set` and `get` methods.